components/automate-ui/src/app/entities/users/user.reducer.spec.ts

```diff
@@ -167,19 +167,19 @@ describe('userStatusEntityReducer', () => {
       });
     });
 
-    describe('UPDATE', () => {
+    describe('UPDATE_NAME_USER', () => {
       const payload = user;
-      const action = new UpdateUser(payload);
+      const action = new UpdateNameUser(payload);
 
       it('sets status to loading', () => {
         const { updateStatus } = userEntityReducer(initialState, action);
         expect(updateStatus).toEqual(EntityStatus.loading);
       });
     });
 
-    describe('UPDATE_SUCCESS', () => {
+    describe('UPDATE_PASSWORD_USER_SUCCESS', () => {
       const payload = { ...user, name: 'test user 123'};
```

Comment:
You've purportedly changed the test to handle a password change, but the payload is still changing the user name. Though the payload is obviously not significant (since the test passes), better to make it clean.